composer.json

```diff
@@ -15,20 +15,20 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "symfony-cmf/routing": "^2.1.0",
-        "symfony/framework-bundle": "^2.8 || ^3.3 || ^4.0",
+        "symfony/framework-bundle": "^3.4 || ^4.0",
         "twig/twig": "^1.35 || ^2.4.4"
     },
     "require-dev": {
         "doctrine/phpcr-odm": "^1.4|^2.0",
-        "symfony/phpunit-bridge": "^4.2.2",
+        "symfony/phpunit-bridge": "^3.4 || ^4.0",
```

Comment:
i locked onto 4.2.2 for the lowest build support. and as this is a dev dependency and does not depend on other symfony things, there is no value in allowing flexibility. we should always go with the latest version.